components/nimbus/src/stateless/cirrus_client.rs

```diff
@@ -22,10 +24,11 @@ use uuid::Uuid;
 /// - `enrolled_feature_config_map`: This field contains the Map representation of the feature value JSON that should be merged with the default feature values.
 /// - `enrollments`: This is the list of ExperimentEnrollments — it should be returned to the client.
 /// - `events`: This is the list of EnrollmentChangeEvents. These events should be recorded to Glean.
-#[derive(Debug, Default, Clone)]
+#[derive(Serialize, Deserialize, Debug, Default, Clone)]
+#[serde(rename_all = "camelCase")]
 pub struct EnrollmentResponse {
-    pub enrolled_feature_config_map: JsonObject,
-    pub enrollments: Vec<ExperimentEnrollment>,
+    pub enrolled_feature_config_map: HashMap<String, JsonObject>,
+    pub enrollments: JsonArray,
```

Comment:
Note to self: this is because we're expecting the enrollments to be stored as an opaque blob, to be passed back in to the SDK, stored, or discarded. 

Perhaps we should make this more clear, by naming them `OpaqueEnrollments` or `OpaqueString`— and not even convert it from a string into a `dict` on the python side.

Reply:
As discussed today I'm going to switch the typings in the request/response types back to the original Rust types, and swap out `JsonObject` for `String` in all the Python-callable code.